content/learning-how-to-teach-coders/quizzes/_index.md

```diff
@@ -1,5 +1,45 @@
 ---
-_db_id: 649
 content_type: topic
 title: Quizzes
----
\ No newline at end of file
+---
+
+## What The Sessions Should Be About
+
+<!-- This type of session involves the leader and follower working through some kind of Q&A type quiz. The topic could be very basic or something more advanced. -->
+
+The session should be about a junior staff members or jtl giving the recruits some questions, quizzes on various topics to work through and see if recruits have a strong and good understanding on the basics foundational skills.
+
+## Purpose
+
+Generally we'll be looking to check or solidify the understanding of a particular topic.
+This can be particularly useful in the context of data science or the more theoretical aspects of software in general (data structures, efficiency, etc).
+
+
+## Procedure
+
+The leader should have a set of questions prepared for the topic being covered or use an online resource. The point isn't to just run through a bunch of questions. We want to make sure the follower really understands the topic being covered. Say we have the question "What's the difference between supervised learning and unsupervised learning", if the follower isn't clear on this, simplify the question and give and example to make it easier for them to understand.
+
+> Lets just define supervised learning or simpler still, machine learning in general.
+
+Even if the follower correctly answers, you should still poke a bit.
+
+> Okay great, that's right. So which project have you completed that uses a supervised model? What are some other examples of supervised models.
+
+The leader should always take the opportunity to engage the follower. Ask follow ups, link it to a particular project, or give context around something. For example mutable vs immutable data structures; once you're sure someone understands the difference between them, you could give context as why certain structures are useful.
+
+Dont ask the follower simple yes and no questions, questions that will make them guess the answer, rather give them something that will make them think, understand and learn from their mistakes. Remember to help the recruit understand and learn, do not give them the answer straight away let them try and find it themselves, rather guide them and point them in the right direction.
+
+If recruits are struggling with some concepts and cannot think of solutions to breaking problems down take them back to the basic concepts and see if they understand everything, if they have a good understanding of the basics then the main focus for them should be to practice and learn problem solving techniques.
+
+## Resources
+- [Python Quizzes](https://realpython.com/quizzes/)
+- [Data science questions](https://github.com/alexeygrigorev/data-science-interviews)
+- [CodeQuizzes](https://www.codequizzes.com/)
+- [JavaScript Quiz](https://www.w3schools.com/js/js_quiz.asp)
+- [Java Quiz](https://www.w3schools.com/java/java_quiz.asp)  
+
+## Practice Problem Solving 
```

Comment:
Are these useful here? Should they be somewhere else? Maybe these problem solving resources would be more appropriate in the  'debugging"  topic (content/learning-how-to-teach-coders/debugging/_index.md).  

These are good resources but I'm not sure that they are good quiz resources. Our syllabus is like code, it should be DRY and Cohesive.